### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by soraphis** [tetra-framework/tetra#42] [General]
Last changes are over a year ago.

I'd really really like to use it, as it is basically exactly how I visioned what I need. But it not only looks dead, but also unfinished (with the big yellow alpha 0.0.4 label on it).

There also is no roadmap.

Why did development stop? is there a more mature alternative that everyone has jumped on, or is the 0.0.4 version working "good enough" for personal use?

**Comment by samwillis** [comment]
Hey @soraphis,

Good questions!

So, I wouldn't characterise it as dead, maybe indefinitely paused?

Is it finished? No unfortunately, and it's very alpha, not production ready.

Why did development stop? Quite simple really, I haven't had time to work on it. I had, and have, many ideas of how I would like to continue building Tetra out, but simply haven't got time to do it.

Maybe one day I will be able to come back to it. Maybe someone else will take the ideas and run with it, or even fork it!

For now I would steer people to using [Django Unicorn](https://www.django-unicorn.com), or a combination of Alpine.js and HTMX.

**Reply by matthiasrsl** [reply]
Hey @samwillis 

I'm also very interested in Tetra, congratulations on this project !

What makes it not production-ready in your opinion ? Is it the lack of features and the lack of API stability given the number of planned features or are there actual security/performance issues that make it unusable ?

**Comment by nerdoc** [comment]
So, after a long time, development will at least go on a bit further. Sam did awesome work to push tetra to an almost perfect starting point, and I'll try my best to get it a bit even better...

So, this topic here can definitely be answered: No, not dead. Was just sleeping a bit.
